roles/pulibrary.tomcat8/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+This role installs Apache Tomcat8
+Requirements
+------------
+
+
+Role Variables
+--------------
+
+Variables can be found in `defaults/main.yml`
+
+Dependencies
+------------
+
+This role depends on `pulibrary.openjdk8`
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables
+passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: roles/pulibrary.tomcat8 }
+
+License
+-------
+
+BSD
```

Comment:
Is this in conflict with https://github.com/pulibrary/princeton_ansible/pull/660/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-df6702ec8636d6df70580f769dfd2d6eR6 and https://github.com/pulibrary/princeton_ansible/pull/660/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-80361ed3024805b172f29ab368897fb5R6?

Reply:
hmmm. the former is correctly in the Tomcat8 Role (which the latter Geoserver one) depends on. It's late in the day and my eyes may be tricking me.